index.html

```diff
@@ -683,115 +681,396 @@
       </h2>
       <section>
         <h2>
-          <code><a>MediaStreamConstraints</a></code> dictionary
```

Comment:
We no longer extend `MediaStreamConstraints` so we can drop that from the Dependencies section.